### PR TITLE
feat(header): move subscribe button

### PR DESCRIPTION
**Pull request by dbeg**
A request came from Erika to change the email subscription button on the developer blog to make it more prominent and similar to that of our [public status page](https://status.widen.com/). This change moves the existing conditional icon up to the header and styles it more like a button.

See temp deploy here: https://buildkite.yden.us/jekyll-theme-guardian/5/_site/index.html ~~(There is some code in this branch that adds a CI build that is from #8, once that is approved and merged I'll rebase and clean this PR up)~~

I realize this change is in the base theme, which means that both the developer and engineering blogs would be affected. Thoughts on this for the engineering blog as well?

@mskelton I quickly hacked some style and positioning, but things are not gracefully responsive at lower viewport widths. Can you take a peek and see if there's a more elegant solution that degrades more gracefully?

### CODE REVIEW

**Comment by mskelton** [comment]
@dbeg I updated the styles to be responsive.  Thanks for the heads up!
Screen Shot 2020-09-01 at 11 55 28 AM

**Comment by dbeg** [comment]
Thanks for the quick edit on that! The background image doesn't gracefully repeat, which might be okay for now. Sometime in the near future we should probably just completely overhaul this theme and start fresh with a v2...

**Comment by mskelton** [comment]
> Sometime in the near future we should probably just completely overhaul this theme and start fresh with a v2...

Love the idea!

**Comment by dbeg** [comment]
@mskelton For the time being, what are your thoughts on dropping the padding on `.site-header` and `.logo` in our responsive breakpoint to tighten things up enough to remove the extra space that shows the repeating background?

**Comment by mskelton** [comment]
@dbeg Updating the background-size to `cover` fixes the responsive issue.  Thanks for calling that out as I missed that before.

File: _sass/_header.scss [hunk 1]
```diff
@@ -2,16 +2,45 @@ header.site-header{
...
+      background-color: #2273c3;
+      border-radius: 4px;
+      color: white;
+      font-size: 10px;
+      font-weight: 400;
+      text-shadow: 0 1px 0 rgba(0, 0, 0, 0.1);
+      text-transform: uppercase;
+      letter-spacing: 2px;
+
+      &:hover {
```
Comment (jweaver-widen) [inline _sass/_header.scss:39]: after checking out in the wild i am not sure it is picking up on the hover effect here. If I may make a suggestion you could do this instead  just outside of the .subscribe block target the hover explicitly with a color hex from patterns 🤙 

.subscribe:hover {
   background-color: #1c5c9c;
}

Comment (jweaver-widen) [reply _sass/_header.scss:39]: Lemme know if that works for ya! 😄 
here is the code pen with the fix: 
https://codepen.io/team/widendev/pen/VwaMMqW

Comment (mskelton) [reply _sass/_header.scss:39]: The url in the description is wrong.  Try this one: https://buildkite.yden.us/jekyll-theme-guardian/7/_site/index.html

Comment (jweaver-widen) [reply _sass/_header.scss:39]: ...lol Well it works 😆

**Review by jweaver-widen — DISMISSED** [review]

**Comment by dbeg** [comment]
I got some feedback from Erika on the button colors - she wanted a bit more contrast, so I inversed the color and background color.

**Review by mskelton — APPROVED** [review]

**Review by forsethc — APPROVED** [review]

**Review by jweaver-widen — APPROVED** [review]
🤙

**Pull request merged**